DXRModules/DeusEx/Classes/DXRStartMap.uc

```diff
@@ -235,16 +248,16 @@ static function StartMapSpecificFlags(FlagBase flagbase, string start_map)
     switch(start_map)
     {
         case "08_NYC_Smug":
-            flagbase.SetBool('KnowsSmugglerPassword',true);
-            flagbase.SetBool('MetSmuggler',true);
+            flagbase.SetBool('KnowsSmugglerPassword',true,,-1);
+            flagbase.SetBool('MetSmuggler',true,,-1);
             break;
         case "11_Paris_Everett":
             //First Toby conversation happened
-            flagbase.SetBool('MeetTobyAtanwe_played',true);
-            flagbase.SetBool('FemJCMeetTobyAtanwe_played',true);
+            flagbase.SetBool('MeetTobyAtanwe_played',true,,-1);
+            flagbase.SetBool('FemJCMeetTobyAtanwe_played',true,,-1);
             break;
         case "14_Vandenberg_Sub":
-            flagbase.SetBool('Ray_dead',true);  //Save Jock!
+            flagbase.SetBool('Ray_dead',true,,-1);  //Save Jock!
```

Comment:
just noticed we don't have a start for area 51, should we?

Reply:
I know we had talked about OceanLab being a ridiculously late start, but with the number of goals we have now, A51 start might be feasible from a bingo perspective (obviously the real end-game is possible too)